data/categories.txt

```diff
@@ -203,7 +203,7 @@ ja:1ガソリンスタンド
 ko:연료|주유소
 mr:पेट्रोल|इंधन|डिझेल
 nb:Drivstoff
-pl:3Benzyna|4Paliwo
+pl:Stacja paliw|3Benzyna|4Paliwo
```

Comment:
Does someone type these words exactly as they are written in the search? If it's a common phrase typed in apps, it's better to add a prefix, eg 4, so it will be displayed as a suggestion. 

If you want to change how gas stations are translated, check types string file. Note that it should match translation in this file.

Reply:
Yes, it is used, like in gmaps as well. Added "4".